lti_consumer/lti_consumer.py

```diff
@@ -925,15 +926,37 @@ def _get_context_for_template(self):
 
         # use bleach defaults. see https://github.com/jsocol/bleach/blob/master/bleach/__init__.py
         # ALLOWED_TAGS are
-        # ['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol',  'strong', 'ul']
+        # ['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol',  'strong', 'ul', 'img']
         #
         # ALLOWED_ATTRIBUTES are
         #     'a': ['href', 'title'],
         #     'abbr': ['title'],
         #     'acronym': ['title'],
-        #
+        #     'img': ['src']
+
         # This lets all plaintext through.
-        sanitized_comment = bleach.clean(self.score_comment)
+        allowed_tags = [
```

Comment:
Do we have a product opinion on this one? that we are adding this limitation on allowed tags?

Reply:
Only new `<img>` tag is added in `allowed_tags` with other tags which were by default allowed by `bleach` and no tag is removed from allowed so it should work as it was previously working + `img`